Frends.FTP.WriteFile/Frends.FTP.WriteFile/WriteFile.cs

```diff
@@ -0,0 +1,174 @@
+﻿using Frends.FTP.WriteFile.Definitions;
+using Frends.FTP.WriteFile.Enums;
+using System;
+using System.ComponentModel;
+using System.IO;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading;
+using FluentFTP;
+
+namespace Frends.FTP.WriteFile;
+
+///<summary>
+/// Main Task class.
+/// </summary>
+public class FTP
+{
+    /// <summary>
+    /// Writes file to the FTP(S) server.
+    /// [Documentation](https://tasks.frends.com/tasks/frends-tasks/Frends.FTP.WriteFile)
+    /// </summary>
+    /// <param name="connection">Connection parameters</param>
+    /// <param name="input">Input parameters</param>
+    /// <param name="cancellationToken">Token to stop task. This is generated by Frends.</param>
+    /// <returns>Object { string Path, double SizeBytes }</returns>
+    public static Result WriteFile([PropertyTab] Connection connection, [PropertyTab] Input input, CancellationToken cancellationToken)

```

Comment:
Please remove unused parameter cancellationToken.

Reply:
Added usage for the cancellationToken.